databooks/common.py

```diff
@@ -21,6 +24,14 @@ def get_logger(name: str, level: str = "INFO") -> logging.Logger:
     return logging.getLogger(name)
 
 
+def set_verbose(logger: logging.Logger) -> None:
```

Comment:
Could you add a oneliner docstring here?

Reply:
Aye captain, BTW if we'd use flake8-docstrings the CI would've blocked it. I added #5 for your consideration.